test/test_worker.rb

```diff
@@ -145,4 +145,17 @@ def call(*args)
       end
     end
   end
+
+  describe '#perform_sync' do
+    class MyCustomWorker2
```

Comment:
Side note: I believe that something is wrong with the test setup above, because when I call it `MyCustomWorker` it fails for a strange reason. I didn't investigate it further.